src/viam/rpc/dial.py

```diff
@@ -277,15 +278,28 @@ def free_str(self, ptr: ctypes.c_void_p):
 
 
 async def dial(address: str, options: Optional[DialOptions] = None) -> ViamChannel:
+    with open("../__init__.py", "r") as file:
+        match = re.search(r'api_version\s*=\s*"([0-9]+\.[0-9]+\.[0-9]+)"', file.read())
+        api_version = match.group(1) if match else None
+
+    with open("../../../pyproject.toml", "r") as file:
+        match = re.search(r'version\s*=\s*"([0-9]+\.[0-9]+\.[0-9]+)"', file.read())
+        sdk_version = match.group(1) if match else None
```

Comment:
For now, I could only find the version in `pyproject.toml` since `__init__.py` technically doesn't have the version in the file itself. Is there a way we can access this variable, something like `viam.__version__`?